GameData/WernhersOldStuff/parts/Aggregat4/A4Engine.cfg

```diff
@@ -161,9 +161,9 @@ PART
 	cost = 190
 	category = Engine
 	subcategory = 0
-	title = Venerable A-4 Rocket Engine
-	manufacturer = Elektromechanische Werke
-	description = Wernher von Kerman's famous war time rocket engine.  It burns 75% ethanol, that keeps liquid at room temperature and is mostly non-corrosive.  The only storage issues relate to the liquid Oxygen oxidizer that needs mild cryogenic storage.  And some of the fuel mix is inevitably stolen by kerbals in need of a good stiff drink.
+	title = AT-VPD "Klassisch" Liquid Fuel Propulsion Engine
+	manufacturer = Atomare Technologien (GmbH)
+	description = Wernher von Kerman's famous war time rocket engine, now manufactured by AtomicTech Inc.'s Keutsch Division.  It burns 75% ethanol, that keeps liquid at room temperature and is mostly non-corrosive.  The only storage issues relate to the liquid oxygen that needs mild cryogenic storage.  Not surprisingly, some of the fuel mix is inevitably stolen by Kerbals in need of a good stiff drink.
```

Comment:
same as agent.

Reply:
Did I make a mistake?

Reply:
not really a mistake @AtomikkuSan, rather changed something that shouldn't be changed due to unintended possible disrespect to original author. In this case the manufacturer `Elektromechanische Werke` shouldn't be radically changed. Agree to adding (GmbH or Ltd.) just not to whole sale change of it.

Reply:
Can we ask TD if we can rename it?

Reply:
> Can we ask TD if we can rename it?

to this? `	title = AT-VPD "Klassisch" Liquid Fuel Propulsion Engine`

that is a good change; and is very much appreciated as it is approved.